examples/simple.py

```diff
@@ -2,13 +2,12 @@
 from benda import bjit, u24
 
 
+@bjit
 def simple() -> u24:
     x = u24(3)
     y = x - u24(2)
     return y
 
 
-if __name__ == "__main__":
-    translated_simple = bjit(simple)
-    print(simple())
-    print(translated_simple())
```

Comment:
This is not that relevant, but open an issue to track implementing being able to do this.
I suspect there is something weird going on anyway for this to not work.

Reply:
Ok